s3fs/core.py

```diff
@@ -89,9 +89,11 @@ class S3FileSystem(object):
         uses the key/secret given, or boto's credential resolver (environment
         variables, config files, EC2 IAM server, in that order)
     key : string (None)
-        If not anonymous, use this key, if specified
+        If not anonymous, use this access key ID, if specified
     secret : string (None)
-        If not anonymous, use this password, if specified
+        If not anonymous, use this secret access key, if specified
```

Comment:
I get that the use of the word "key" twice is a bit confusing, but unfortunately that's what AWS refers to them as. From http://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys:

> Access keys consist of an access key ID (for example, AKIAIOSFODNN7EXAMPLE) and a secret access key (for example, wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY)

That's also what they're called in [boto](http://boto3.readthedocs.io/en/latest/guide/configuration.html#configuring-credentials).

I think this wording in the documentation will make sense to people with familiarity with AWS credentials.